pageserver/src/tenant.rs

```diff
@@ -6590,13 +6578,15 @@ mod tests {
 
         // We create one bottom-most image layer, a delta layer D1 crossing the GC horizon, D2 below the horizon, and D3 above the horizon.
         //
-        //  | D1 |                       | D3 |
+        //                             | D3 |
+        //  | D1 |
         // -|    |-- gc horizon -----------------
         //  |    |                | D2 |
         // --------- img layer ------------------
         //
         // What we should expact from this compaction is:
-        //  | Part of D1 |               | D3 |
+        //                             | D3 |
+        //  | Part of D1 |
         // --------- img layer with D1+D2 at GC horizon------------------
```

Comment:
With the new forced lsn range, D2 is LSN range `0x20..0x48` and thus crossing the gc horizon.

Not sure hot represent in ASCII that all its _values_ are below the GC horizon.

I guess best just explain it in text, like in my suggestion below.

Also, while typing it out, maybe it's worth having a 4th delta D4 in the game that is completely below GC horizon?


```suggestion

        // We create
        // - one bottom-most image layer,
        // - a delta layer D1 crossing the GC horizon with data below and above the horizon,
        // - a delta layer D2 crossing the GC horizon with data only below the horizon,
        // - a delta layer D3 above the horizon.
        //
        //                             | D3 |
        //  | D1 |
        // -|    |-- gc horizon -----------------
        //  |    |                | D2 |
        // --------- img layer ------------------
        //
        // What we should expact from this compaction is:
        //                             | D3 |
        //  | Part of D1 |
        // --------- img layer with D1+D2 at GC horizon------------------
```

Reply:
Thanks! This looks more clear.